examples/basic/variables.tf

```diff
@@ -0,0 +1,122 @@
+##############################################################################
+# Input Variables
```

Comment:
An example should not expose all these variables. just the very bare minimum, which would be these -> https://github.com/terraform-ibm-modules/terraform-ibm-module-template/blob/main/examples/basic/variables.tf

You can hard code values in the main.tf of the example since its not designed to be some flexible solution like a DA, its just a reference example.